### PR TITLE
cmd/action: add schema flag

### PR DESCRIPTION
**Pull request by yonidavidson**

### CODE REVIEW

**Review by rotemtam — COMMENTED** [review]
Looks good. Can you add tests?

File: cmd/action/inspect.go [hunk 1]
```diff
@@ -46,6 +48,7 @@ func init() {
 	)
 	InspectCmd.Flags().BoolVarP(&InspectFlags.Web, "web", "w", false, "Open in a local Atlas UI")
 	InspectCmd.Flags().StringVarP(&InspectFlags.Addr, "addr", "", "127.0.0.1:5800", "used with -w, local address to bind the server to")
+	InspectCmd.Flags().StringSliceVarP(&InspectFlags.Schema, "schema", "s", nil, "Set schema name")
```
[the remark this answers is not in the document]
Comment (yonidavidson) [reply cmd/action/inspect.go]: Fixed

File: cmd/action/mux.go [hunk 2]
```diff
@@ -102,9 +102,7 @@ func SchemaNameFromDSN(url string) (string, error) {
 func postgresSchema(dsn string) (string, error) {
 	url, err := url.Parse(dsn)
 	if err != nil {
-		// For backwards compatibility, we default to "public" when failing to
-		// parse.
-		return "public", nil
+		return "", nil
 	}
```
Comment (a8m) [inline cmd/action/mux.go:106]: Why did you remove this?

Comment (yonidavidson) [reply cmd/action/mux.go:106]: no need anymore, now that we inspect the realm there is no need to define the "default schema" to inspect ( it works)

**Review by rotemtam — COMMENTED** [review]
Looks good. See failing tests
